.travis.yml

```diff
@@ -24,10 +24,10 @@ env:
 matrix:
     include:
         - php: 7.1
-        - php: 7.1
-          env: deps=high
         - php: 7.2
-          env: deps=low
+          env: deps=high
+        - php: 7.3
+          env: deps=high
```

Comment:
Looks like we don't have test for `deps=low` anymore.

Reply:
which envs do you want to test? I enabled all in next commit but that seems overkill